File: atinject-api/src/main/java/org/atinject/api/user/dto/UpdateUserNameRequest.java
package org.atinject.api.user.dto;

import org.atinject.core.websocket.dto.WebSocketRequest;

public class UpdateUserNameRequest extends WebSocketRequest {

	private static final long serialVersionUID = 1L;
	
	private String name;

	public String getName() {
		return name;
	}

	public UpdateUserNameRequest setName(String name) {
		this.name = name;
		return this;
	}
	
	
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/user/UpdateUserNameRequest.js
define(["atinject/WebSocketRequest"],
function(WebSocketRequest){
    function UpdateUserNameRequest() {
        this._class = "org.atinject.api.user.dto.UpdateUserNameRequest"
    };

    UpdateUserNameRequest.prototype = new WebSocketRequest();

        UpdateUserNameRequest.prototype.getName = function() {
        return this.name;
    };

    UpdateUserNameRequest.prototype.setName = function(name) {
        this.name = name;
        return this;
    };


    
    return UpdateUserNameRequest;
});
